central/apitoken/testutils/testutils.go

```diff
@@ -0,0 +1,31 @@
+package testutils
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/protocompat"
+	"github.com/stackrox/rox/pkg/uuid"
+)
+
+func GenerateToken(t *testing.T, now *time.Time, expiration *time.Time, revoked bool) *storage.TokenMetadata {
```

Comment:
Not sure if nil time pointers are ever used and really needed.

Reply:
removed pointers